README.md

```diff
@@ -0,0 +1,20 @@
+# [CollabCamp website](http://collab.camp/)
+
+using [Jekyll](http://jekyllrb.com/)
+
+CollabCamp events bring together social entrepreneurs, sharers, hackers, government officials and citizens to collaboratively work on projects which benefit the collaborative economy (sharing). CollabCamp events are free to attend. We celebrate collaboration and sharing. Let's get it done! 
+
+### Editing this site locally
+
+To clone and run a server locally for testing:
+
+    git clone https://github.com/collabcamp/collabcamp.github.io.git
+    cd collabcamp.github.io
+    sudo npm install jekyll
```

Comment:
@wvengen  bundle install returns
$ bundle install
Bundler::GemfileNotFound

I'll remove the sudo now